modules/apps/site/site-insurance-site-initializer/src/main/zippableResources/fragments/raylife/fragments/get-in-touch/index.css

```diff
@@ -0,0 +1,67 @@
+h1, h3 {
+	font-weight: var(--display1-weight);
+}
+
+p {
+	color: var(--gray-700);
```

Comment:
All this file `index.css` must be indented, because we have a mix of spaces here